instructor/ui/instructor.js

```diff
@@ -751,8 +842,315 @@ $(function() {
+          type: "POST",
+          data: data
+      })
+  }
+  /* Fluorecent image selecting workflow support automation Image Set creation */
+  function createNewSet(selectedIds, target) {
+      var selectedIdsList = selectedIds.split(':');
+      moveSelected(selectedIdsList.pop(), true, target);
+      saveSelectedImages($('.scb_ab_f_save_image').data('group_id'));
+      $.each(selectedIdsList, function (_, id) {
+          $.ajax({
+              url: '/ab/assignments/add_new_image_set/',
+              type: "POST",
+              data: {mapping_id: $('.scb_ab_f_save_image').data('pk')}
+          }).then(function (newGroupId) {
+              $(".scb_ab_s_small_image_selected").removeClass("scb_ab_s_small_image_selected");
+              $(target).empty();
+              moveSelected(id, true, target);
+              saveSelectedImages(newGroupId);
+          })
+      });
+      location.reload();
+  }
+
+  /* Dragging selected images between Selected images and Image bank */
+  function dragSelected(event, direction=true) {
+      var selectedId = event.originalEvent.dataTransfer.getData("text");
+      if (selectedId.indexOf(":") > -1) {
+          if ($(".scb_ab_f_save_image").data("group_id") && direction && $(event.target).hasClass('scb_f_image_filter')) {
+              return $(".scb_ab_s_small_image_selected").length ? null : createNewSet(selectedId, event.target);
+          } else {
+              $.each(selectedId.split(':'), function (_, id) {
+                  moveSelected(id, direction, target=event.target);
+              });
+          };
+      } else {
+          moveSelected(selectedId, direction, target=event.target);
+      }
+      unselectImages();
+  }
+
+  /* Event handler for drag&drop selection workflow */
+
+  // On draging one image drag all selected images
+  $('.scb_ab_f_select_image').on('dragstart', function (e) {
+      e.originalEvent.dataTransfer.setData("text", dragStartSelection(e));
+  });
+
+  function dragStartSelection(event) {
+      var selectedImages = $(".scb_ab_s_image_selected"),
+          imageIdList = [];
+      if (selectedImages.length > 0) {
+          selectedImages.each(function () {
+              imageIdList.push($(this).attr("id"))
+          });
+          return imageIdList.join(":")
+      } else {
+          return event.target.id
+      }
+  }
+
+  $filesSelect.on('drag dragend dragover dragenter dragleave drop', function(e) {
+    e.preventDefault();
+    e.stopPropagation();
+  })
+  .on('dragstart', function (e) {
+      e.originalEvent.dataTransfer.setData("text", dragStartSelection(e));
+  })
+  .on('dragover dragenter', function () {
+    $filesSelect.addClass('is-dragover');
+  })
+  .on('dragleave dragend drop', function () {
+    $filesSelect.removeClass('is-dragover');
+  })
+  .on('drop', dragSelected);
+
+  // Handler for the form submitting process.
+  $imageForm.on('submit', function(e) {
+      if ($imageForm.hasClass('is-uploading')) return false;
```

Comment:
Please enclose `return false` in braces.

Reply:
fixed